### PR TITLE
Update README.md to add Rhea biped

### PR DESCRIPTION
**Pull request by vmc-7645**
Added Rhea bipedal robot to list of bipeds.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,6 +18,7 @@ A curated list of awesome robots that can be fully built at home and operated us
 | Name  | Type          | License    | Model | Software |
 |-------|---------------|------------|-------|----------|
 | [Upkie](https://hackaday.io/project/185729-upkie-wheeled-biped-robot) | Wheeled biped | Apache-2.0 | [URDF](https://github.com/tasts-robots/upkie_description) | [GitHub](https://github.com/tasts-robots) |
+| [Rhea](https://github.com/G-Levine/rhea_description) | Wheeled biped | Apache 2.0 | [URDF](https://github.com/tasts-robots/upkie_description) | [GitHub](https://github.com/tasts-robots/) |
```
Comment (stephane-caron) [inline README.md:21]: The URDF and GitHub links should be updated as well.

**Comment by stephane-caron** [comment]
I'd be delighted to add Rhea to this list! However I think its motion control software is not open source for now (@G-Levine correct me if I'm wrong) so not within the scope of this particular list (*c.f.* [this guideline](https://github.com/tasts-robots/awesome-open-source-robots/blob/main/CONTRIBUTING.md#-open-motion-control)).

Note that Rhea is already in the related [Awesome Robot Descriptions](https://github.com/robot-descriptions/awesome-robot-descriptions#bipeds) list.

**Comment by G-Levine** [comment]
In response to your question @stephane-caron, I’ve published the motion control code for Rhea [here](https://github.com/G-Levine/Rhea), but I haven’t decided on a license for it yet.

**Comment by stephane-caron** [comment]
Thanks @G-Levine, I'll be watching that repository. If you're able to give it a permissive or copyleft license I'd be delighted to add Rhea to this list too :smiley: 

(Also this made me realize the same question was left open for the Slab robot: updated the list and moved the discussion to [a separate PR](https://github.com/tasts-robots/awesome-open-source-robots/pull/2).)

**Pull request closed without merging**